File: 백준/Bronze/1598. 꼬리를 무는 숫자 나열/꼬리를 무는 숫자 나열.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt() - 1;
        int b = sc.nextInt() - 1;
        
        int result = Math.abs(a / 4 - b / 4) + Math.abs(a % 4 - b % 4);
        System.out.println(result);

        sc.close();
    }
}

File: 백준/Bronze/1598. 꼬리를 무는 숫자 나열/꼬리를 무는 숫자 나열.cs
string[] input = Console.ReadLine().Split();
int a = int.Parse(input[0]) - 1;
int b = int.Parse(input[1]) - 1;

int result = Math.Abs(a / 4 - b / 4) + Math.Abs(a % 4 - b % 4);
Console.WriteLine(result);